deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -89,6 +89,22 @@ kube-prometheus-stack:
     # tls:
     #   enabled: false
 
+  ## Component scraping kube state metrics
+  ##
+  kubeStateMetrics:
+    serviceMonitor:
+      # relabel configs to apply to samples before ingestion.
+      ##
+      # - sourceLabels: [__meta_kubernetes_pod_node_name]
+      #   separator: ;
+      #   regex: ^(.*)$
+      #   targetLabel: nodename
+      #   replacement: $1
+      #   action: replace
+      relabelings:
```

Comment:
How should we configure this for installing openebs-monitoring in custom Prometheus stack if `servicemonitor` for `kube-state-metrics` is not installed?

Reply:
This is not necessary. Without this also will work. I'll be removing this.

Reply:
This is not required and will be removing it.